pkg/controller.v1/common/job.go

```diff
@@ -275,7 +275,7 @@ func (jc *JobController) ReconcileJobs(
 					pg = volcanoPodGroup
 					return nil
 				}
-			case GangSchedulerSchedulerPlugins:
+			case GangSchedulerSchedulerPlugins, GangSchedulerKoordinator:
```

Comment:
why is the PR bigger than just this line?

if we can have this, I would support this change. Otherwise, I'm opposed.

```golang
switch jc.Config.GangScheduling {
case GangSchedulerVolcano:
  ...
default:
}
```

Reply:
here, I'm considering scheduling-plugins the "default" because it has a higher chance of becoming part of upstream kubernetes at some point.